doc/source/index.rst

```diff
@@ -13,11 +13,44 @@ built-in Python data types and numpy arrays.
 Both simple and complex searches are supported. Convenience fucntions
 provide a simple way to handle tabular data and image sequences.
 
-See the :ref:`api` for details.
+Examples of Basic Usage
+-----------------------
+
+If they are not already imported, import ``DataBroker`` and its convenience
+fucntions. (This step may already be done as part of an
+`IPython profile <http://nsls-ii.github.io/beamline-configuration.html#ipython-profiles>`_.)
+
+.. code-block:: python
+
+    from databroker import DataBroker as db, get_table, get_images, get_events
+
+Example 1: Load scalar data in a table.
```

Comment:
I appreciate the simplicity of the example.  Do you want to mention the `fill` kwarg which defaults to filling in external data? Maybe change the example to `get_table(last_run, fill=False)`?